test/source/mock/wkd/wkd-endpoints.ts

```diff
@@ -116,6 +206,9 @@ export const mockWkdEndpoints: HandlersDefinition = {
   '/.well-known/openpgpkey/localhost/hu/pob4adi8roqdsmtmxikx68pi6ij35oca?l=incorrect': async () => {
     return alice; // advanced for incorrect@localhost
   },
+  '/.well-known/openpgpkey/localhost/hu/66iu18j7mk6hod4wqzf6qd37u6wejx4y?l=some.revoked': async () => {
+    return validAmongRevoked;
```

Comment:
I just noticed - we are returning armored keys here on the WKD endpoints but that does not reflect how WKD returns keys (which should be unarmored bytes, concatenated). I'll make another issue for this.